Cargo.toml

```diff
@@ -52,6 +52,7 @@ minidom = "0.12"
 minidom_ext = "1"
 minidom_writer = "1"
 num-traits = "0.2"
+once_cell = "=1.0.0-pre.2"
```

Comment:
Probably gonna need to remove that before merging. Apart from that, I'm approving this message.

Reply:
same